wgpu-core/build.rs

```diff
@@ -0,0 +1,15 @@
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases::cfg_aliases! {
```

Comment:
I'm a bit concerned about introducing a build script here. However, I do agree that using `cfg` in the code is much much nicer, so this seems like a good change to do.

Reply:
Yeah it does add a build script but `cfg_aliases` is just a `macro_rules!` macro with no deps so it shouldn't increase build times by a noticeable amount and it also only resolves to simple `println!` statements that are interpreted by Cargo so it isn't dependent on any host environment or anything like that either.